### ISSUE TITLE
[BUG] Accessibility: Admiralty Logo fails WCAG 1.1.1, 2.5.3

### ISSUE THREAD

**Issue by LukeTaylor34** [UKHO/admiralty-design-system#130]
**Describe the bug**
The ALT description for the Admiralty logo is currently "Admiralty Stacked Logo" which does not sufficiently describe its purpose or include all the text visible in the image.

**Expected behavior**
My thoughts are the preferred solution would be to change the Admiralty Logo and UK Hydrographic Office Logo to be link elements with either 'Admiralty Maritime Data Solutions' or 'UK Hydrographic Office' as its text. They can be styled to look as they currently do with CSS and any icons being an SVG element that's non-focusable.

But at a minimum to pass WCAG the ALT text for the Admiralty logo will need to be change to 'Admiralty Maritime Data Solutions | UK Hydrographic Office'. As there are multiple links to the same destination on a page, Admiralty Logo in the header and UK Hydrographic Office Logo in footer they should have the same description.

**Screenshots**
![image](https://github.com/UKHO/admiralty-design-system/assets/97240814/3c81eb54-8c2f-47b5-a5b9-7aa227937736)

**Additional context**
Level A 1.1.1 - All non-text content that is presented to the user has a text alternative that serves the equivalent purpose.
F30: Failure of Success Criterion 1.1.1 and 1.2.1 due to using text alternatives that are not alternatives (e.g., filenames or placeholder text)
Level A 2.5.3 - If the non-text content is an image of text, then the name contains the text that is presented visually.
F96: Failure due to the accessible name not containing the visible label text.

**Comment by KatiePUX** [comment]
although we accept this is an issue, are hands are tied due to marketing constraints at this time

**Comment by KatiePUX** [comment]
KP discussed with LT - this is just to change the ALT text so should not be an issue - if not we should escalate to Bruce as this is an accessibility requirement so something does need to change.

**Comment by UKHOGreg** [comment]
> KP discussed with LT - this is just to change the ALT text so should not be an issue - if not we should escalate to Bruce as this is an accessibility requirement so something does need to change.

@KatiePUX what do we want the alt text to say?
Luke kindly suggested 'Admiralty Maritime Data Solutions' or 'UK Hydrographic Office'

Our Logo just says ADMIRALTY now. Should our alt text just say 'ADMIRALTY'?

**Comment by UKHOGreg** [comment]
I've just noticed:
> Level A 2.5.3 - If the non-text content is an image of text, then the name contains the text that is presented visually.
F96: Failure due to the accessible name not containing the visible label text.

In Luke's message, guessing that means it must say ADMIRALTY then.

**Comment by KatiePUX** [comment]
Yeah I agree with you, lets not complicate it - say what is seen! Shall we just confirm in the morning at SU?

**Comment by UKHOGreg** [comment]
Will have to be Tuesday for me @KatiePUX !
